index.src.html

```diff
@@ -805,6 +943,30 @@ <h4 id="cors-preflight">CORS preflight</h4>
           1.  <var ignore>entry</var>'s [=cache entry/IP address space=] is
               equal to |request|'s [=request/target IP address space=].
 
+  <h4 id="fetch-api">Fetch API</h4>
+
+  The Fetch API needs to be adjusted as well.
+
+  - Append an optional [=map/entry=] to {{RequestInfo}}, whose [=map/key=] is
+    <dfn export>targetAddressSpace</dfn>, and [=map/value=] is a
+    {{RequestTargetAddressSpace}}.
+      <pre class="idl">
+        dictionary RequestInit {
+          // ...
+          RequestTargetAddressSpace targetAddressSpace;
+        };
+      </pre>
+  - Define {{RequestTargetAddressSpace}} as follows.
+      <pre class="idl">
+        enum RequestTargetAddressSpace { "private", "local" };
+      </pre>
+  - The <a constructor for=Request lt="Request(input, init)"><code>new
+    Request(<var ignore=''>input</var>, |init|)</code></a> is
+    appended with the following step right before setting [=this=]'s [=request=]
+    to |request|:
+    1.  If |init|["{{RequestInit/targetAddressSpace}}"] exists, set |request|'s
+        [=target IP address space=] to it.
```

Comment:
We should convert the value here - in IDL it's a string, we want `[=IP address space/local=]` or `[=IP address space/private=]`.